### PR TITLE
docs(store): remove reducer wrapper function

### LINKED ISSUE #3352: Docs: remove extra exported function from reducer file in Getting Started
The exported function is no longer needed with Ivy. We can export the reducer from `createReducer` directly.

BEFORE:

```ts
const _counterReducer = createReducer(
  initialState,
  on(increment, (state) => state + 1),
  on(decrement, (state) => state - 1),
  on(reset, (state) => 0)
);
 
export function counterReducer(state, action) {
  return _counterReducer(state, action);
}
```

The counter file is located here: 

https://github.com/ngrx/platform/blob/master/projects/ngrx.io/content/examples/store/src/app/counter.reducer.ts

## I would be willing to submit a PR for the docs :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

### PR DESCRIPTION
**Pull request by Gopans**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #3352 

## What is the new behavior?

The reducer wrapper function is removed from the store example.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/examples/store/src/app/counter.reducer.ts [hunk 1]
```diff
@@ -4,13 +4,10 @@ import { increment, decrement, reset } from './counter.actions';
 
 export const initialState = 0;
 
-const _counterReducer = createReducer(
+export const _counterReducer = createReducer(
```
Comment (santoshyadavdev) [inline projects/ngrx.io/content/examples/store/src/app/counter.reducer.ts:7]: You need to remove _ from function name too

**Review by santoshyadavdev — CHANGES_REQUESTED** [review]
Small change in function name

**Review by markostanimirovic — APPROVED** [review]

File: projects/ngrx.io/content/examples/store/src/app/counter.reducer.ts (hunk 1 above)
Comment (santoshyadavdev) [reply projects/ngrx.io/content/examples/store/src/app/counter.reducer.ts]: Thank you

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks @Gopans!

**Pull request merged**